packages/client/src/api/leaderBoard/index.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseApi } from '@api/baseApi'
+import {
+    LeaderBoardDTO,
+    LeaderboardTableDTO,
+    LeaderboardTableResponse,
+} from './types'
+
+export class LeaderBoardApi extends BaseApi {
+    constructor() {
+        super('leaderboard')
+    }
+
+    saveRecord = async (leaderboardData: LeaderBoardDTO) => {
+        const { data } = await this.client.post<string>('', leaderboardData)
```

Comment:
Я влил свои изменения, поэтому придется теперь так писать)

```suggestion
        const { data } = await this.client.post<string>(this.withUrl(), leaderboardData)
```